src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -953,15 +953,15 @@ void ReconstructionEngine_sequentialSfM::exportStatistics(double reconstructionT
         std::map<std::size_t, std::size_t> obsHistogram;
         for (const auto& iterTracks : _sfmData.getLandmarks())
         {
-            const Observations& obs = iterTracks.second.observations;
+            const Observations& obs = iterTracks.second.getObservations();
             if (obsHistogram.count(obs.size()))
                 obsHistogram[obs.size()]++;
             else
                 obsHistogram[obs.size()] = 1;
         }
 
         for (std::size_t i = 2; i < obsHistogram.size(); ++i)
-            _jsonLogTree.add("sfm.observationsHistogram." + std::to_string(i), obsHistogram[i]);
+            _jsonLogTree.add("sfm.getObservations()Histogram." + std::to_string(i), obsHistogram[i]);
```

Comment:
There's been a faulty find & replace here: we should keep `observationsHistogram` as the name.

Reply:
Fixed in c8d57ce60eeb422e3c1968989134f06d0566f5fc.